rack-protection/lib/rack/protection/base.rb

```diff
@@ -8,6 +8,27 @@
 
 module Rack
   module Protection
+    ##
+    # Base class for middlewares provided by rack-protection.
+    #
+    # == Options
+    #
+    # These may be given to any subclass, however they may affect different subclasses differently.
+    #
+    # [<tt>:reaction</tt>] - name of a method of the class that will be used to respond when the
+    #                        middleware rejects the response. Default behavior is to use #default_reaction.
+    # [<tt>:logging</tt>] - if true, debug and warning statements are sent to the logger.
+    # [<tt>:message</tt>] - if set, and <tt>:reaction</tt> has not been set, this is the
+    #                       text of the message sent to the client when the request is rejected. See DEFAULT_OPTIONS.
+    # [<tt>:session_key</tt>] - string to use as the key into <tt>env</tt> where the current
+    #                           session is stored. See DEFAULT_OPTIONS.
+    # [<tt>:status</tt>] - HTTP status to use when a request is rejected.
+    #                      See DEFAULT_OPTIONS.
```

Comment:
Maybe?

```suggestion
    # [<tt>:status</tt>] - HTTP status to use when a request is rejected with the <tt>deny</tt> reaction.
    #                      See DEFAULT_OPTIONS.
```

Reply:
Made a similar change in 0e082e88c5102bffb674b1d755256c1537321c67